src/types/reduxTypes/eventsStateTypes.ts

```diff
@@ -0,0 +1,24 @@
+export interface IEventsProps {
+  events: IEvents
+}
+
+export interface IEvents {
+  isEventsLoading: boolean
+  error: string | null
+  eventsData: IEventsDataProps[]
```

Comment:
If it will we events list change name to eventsList

Reply:
done